### ISSUE TITLE
dataset problem

### ISSUE THREAD

**Issue by forkliu0826** [ShichaoSun/SS4Sum#1]
Hello author, I am interested in your article and I want to run your code on other data sets, so I would like to know how to do data preprocessing？

**Comment by forkliu0826** [comment]
Hello author, I am interested in your article and I want to run your code on other data sets, so I would like to know how to do data preprocessing

**Comment by ShichaoSun** [comment]
> Hello author, I am interested in your article and I want to run your code on other data sets, so I would like to know how to do data preprocessing

Please follow the process of https://github.com/mswellhao/PacSum.

I use h5df format file, where the dataset is a list and each item is a document/article (with summary in test split). Note that the document/article is a list of sentences.

**Comment by forkliu0826** [comment]
Thank you for your answer. I have another question, that is, where can I get the papers or codes of the experiments based on the algorithms you use to compare experiments, such as TF-IDF,BERT, etc., and run unsupervised extraction abstracts

**Comment by forkliu0826** [comment]
I found that the experimental results in your paper are different from those in pacsum's paper. I would like to ask you about the differences in the following dataset construction

**Comment by ShichaoSun** [comment]
Hi, 
Sorry, I am not very sure where the difference is. Can you provide more details about the difference?

I guess that the sight difference between textrank (bert) with ours ( Pacsum BERT) may originate from the random seed.  Original BERT with textrank is not shown in PacSum paper.

For processing dataser, I only  remove some too short items of training dataset. I do nothing for testing dataset.

**Comment by forkliu0826** [comment]
Sorry, there may be some problem in my expression, a very simple problem. For example, in this article by PacSum, the result of PacSum Bert on the NYT data set R1,R2 and RL is 41.4/21.7/37.5. Why is the result in your article 30.99/10.28/26.81? What is the reason for the huge gap in results？

**Comment by ShichaoSun** [comment]
Hi,
This paper focuses on the effect of sentence embedding, and we use textrank as our backbone method (ignoring the effect of position), so our result is the textrank(bert) in PacSum.

**Comment by forkliu0826** [comment]
Sorry, what I mean is that the results of PacSum Bert in NYT dataset R1, R2 and RL in the experiment of the original paper are 41.4/21.7/37.5. Why is the result of NYT dataset replicated by PacSum Bert in your article 30.99/10.28/26.81

**Comment by ShichaoSun** [comment]
PACSum Bert with pac algorithm is 41.4/21.7/37.5. In my paper, I ignore the position effect and only use textrank to compare the effect of sentence embedding. As I mention, I reported the textrank(bert) in PacSum as PACSum bert in my paper.
